prime-router/src/main/kotlin/history/azure/DeliveryFacade.kt

```diff
@@ -54,6 +59,13 @@ class DeliveryFacade(
             "End date must be after start date."
         }
 
+        var reportId: UUID? = null
+        try {
+            reportId = if (reportIdStr != null) UUID.fromString(reportIdStr) else null
```

Comment:
This doesn't seem quite to me?  At least testing locally, if I pass in an invalid UUID I now just get all the results?

Reply:
fixed

Reply:
Nice!  Just tested it out and I got a 400 message with the invalid ID in it